api/proto/teleport/workloadidentity/v1/resource.proto

```diff
@@ -133,12 +134,26 @@ message WorkloadIdentitySPIFFEX509 {
   //
   // If not provided, the X509-SVID will be issued with an empty Subject DN.
   X509DistinguishedNameTemplate subject_template = 2;
+  // Control the maximum TTL of X509-SVIDs issued using this WorkloadIdentity.
+  //
+  // If a X509-SVID is requested with a TTL greater than this value, then the
+  // returned X509-SVID will have a TTL of this value.
+  //
+  // Defaults to 24 hours. The maximum this value can be set to is 14 days.
```

Comment:
For JWT, the max is 24 hours.
For X.509 certs, the max is 14 days.

Any reason to have those two TTLs being so different?

Also, would it make sense to add a max TTL at the `WorkloadIdentitySPIFFE`?

Reply:
> Any reason to have those two TTLs being so different?

Mostly that JWT-SVIDs are much less secure - I debated having the same limit but I was concerned that it was too much of a footgun. If a customer comes along with a convincing use-case, I'm definitely open to raising the limit.

> Also, would it make sense to add a max TTL at the WorkloadIdentitySPIFFE?

I considered this but I think that breaking it down per type better respects their different security characteristics. If they both had the same max max TTL, then I'd definitely consider it.

Reply:
Sounds good 👍